### DISCUSSION TITLE
Blender

### DISCUSSION THREAD

**Discussion by AlfredOhNO** [catppuccin/catppuccin#1944] [Port Requests]
**Are you available to help in the creation of this port? 😄 (optional)**
Yes

**Additional context 📷 (optional)**
I've already made a Catppuccin-pink.xml and a cholorscheme agnostic Catppuccin.xml that covers every color I can see, though I haven't thoroughly tested it
![example](https://cdn.discordapp.com/attachments/573966960702914560/995508568256167996/unknown.png)
[Catppuccin-pink.zip](https://github.com/catppuccin/catppuccin/files/9079560/Catppuccin-pink.zip)
[Catppuccin.zip](https://github.com/catppuccin/catppuccin/files/9079561/Catppuccin.zip)

**Comment by nekowinston** [comment]
Oh very cool, I also worked on this! I'd be obliged if you check out my version as well
https://github.com/nekowinston/ctp-blender

![Blender theme](https://cdn.discordapp.com/attachments/919570286754529330/986720032518197328/unknown.png)

**Comment by AlfredOhNO** [comment]
Looks nice! probably a bit more theme agnostic than mine lol

**Comment by nekowinston** [comment]
Yeah the way I approached the port was super lazy tbh, I just made the existing dark default theme match Catppuccin colours automatically, so it's likely that certain colours/buttons won't look nice.

I had the best result when matching against `frappe` colours, then using [`puccinier`](https://github.com/catppuccin/toolbox#-puccinier) to port it to the other flavours.

**Comment by AlfredOhNO** [comment]
Right, I just manually did a find and replace in the xml theme file (using a text editor) and matched it to the Mocha pallette, followed by double checking the interface and theme settings within blender to make sure it worked.
Messy and slow, but functional.

**Comment by AlfredOhNO** [comment]
I have since done some minor tweaking to the pink version, and added a more color agnostic 'Catppuccin.xml' to the original post (https://github.com/catppuccin/catppuccin/files/9079561/Catppuccin.zip). Should be a bit more polished now. (I replaced pink with the 'overlay2' color from the Mocha palette.)

**Comment by sgoudham** [comment]
Closing as #2303 has been raised, along with #2014 and #2040
